spec/validation/checksums.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+require "spec_helper"
+require_relative "../../lib/ht_sip_validator/sip/sip"
+require_relative "../../lib/ht_sip_validator/validation/checksums"
```

Comment:
Should not be necessary; `spec_helper` is already requiring `ht_sip_validator` which should ultimately pulls in everything. I have seen that considered a little bit of an anti-pattern in that it can slow down tests if you aren't running all of them, but these tests are fast enough right now that I'm not really worried about it!